vector/src/main/java/im/vector/app/features/home/room/detail/timeline/helper/VoiceMessagePlaybackTracker.kt

```diff
@@ -125,4 +136,8 @@ class VoiceMessagePlaybackTracker @Inject constructor() {
             data class Recording(val amplitudeList: List<Int>) : State()
         }
     }
+
+    fun interface ActivityListener {
+        fun onUpdate(isActive: Boolean)
```

Comment:
Nit: `isActive` is not really accurate to me. Maybe rename to `isPlayingOrRecording`? Also for the `val` at line 64.
This is not a big deal though, I can live with it :)

Reply:
makes sense, will update!